website/project/views/drafts.py

```diff
@@ -197,57 +194,6 @@ def draft_before_register_page(auth, node, draft, *args, **kwargs):
     return ret
 
 
-@must_have_permission(ADMIN)
-@must_be_branched_from_node
-@http_error_if_disk_saving_mode
-def register_draft_registration(auth, node, draft, *args, **kwargs):
-    """Initiate a registration from a draft registration
-
-    :return: success message; url to registrations page
-    :rtype: dict
-    """
-    json_data = request.get_json()
-    if 'data' not in json_data:
-        raise HTTPError(http.BAD_REQUEST, data=dict(message_long='Payload must include "data".'))
-    data = json_data['data']
-    if 'attributes' not in data:
-        raise HTTPError(http.BAD_REQUEST, data=dict(message_long='Payload must include "data/attributes".'))
-    attributes = data['attributes']
-    registration_choice = attributes['registration_choice']
-    validate_registration_choice(registration_choice)
-
-    # Don't allow resubmission unless submission was rejected
-    if draft.approval and draft.approval.state != Sanction.REJECTED:
-        raise HTTPError(http.CONFLICT, data=dict(message_long='Cannot resubmit previously submitted draft.'))
-
-    register = draft.register(auth)
-    draft.save()
-
-    if registration_choice == 'embargo':
-        # Initiate embargo
-        embargo_end_date = parse_date(attributes['lift_embargo'], ignoretz=True).replace(tzinfo=pytz.utc)
-        try:
-            register.embargo_registration(auth.user, embargo_end_date)
-        except ValidationError as err:
-            raise HTTPError(http.BAD_REQUEST, data=dict(message_long=err.message))
-    else:
-        try:
-            register.require_approval(auth.user)
-        except NodeStateError as err:
-            raise HTTPError(http.BAD_REQUEST, data=dict(message_long=str(err)))
-
-    register.save()
-    push_status_message(language.AFTER_REGISTER_ARCHIVING,
-                        kind='info',
-                        trust=False,
-                        id='registration_archiving')
-    return {
-        'status': 'initiated',
-        'urls': {
-            'registrations': node.web_url_for('node_registrations', _guid=True)
-        }
-    }, http.ACCEPTED
-
```

Comment:
If you're getting rid of waffle stuff too, seems like there's a bit more stuff to get rid of in this file including the `submit_draft_for_review` view (maybe more?)

Reply:
I wanted to avoid removing Prereg stuff like `submit_draft_for_review`,  until we can totally get rid of it, . Though technically it's both an old v1 view and a Prereg it's probably better to remove all the prereg stuff in another ticket. I have a stash that removes most obsolute prereg stuff, but there's some general sanctions related code I was unsure about so I decided to make this strictly old v1 routes.